src/lib/services/email-service.ts

```diff
@@ -31,6 +31,8 @@ export interface IEmailEnvelope {
 
 const RESET_MAIL_SUBJECT = 'Unleash - Reset your password';
 const GETTING_STARTED_SUBJECT = 'Welcome to Unleash';
+const SCHEDULED_CHANGE_CONFLICT_SUBJECT =
+    'Unleash - Detected some conflicts with your scheduled changes';
```

Comment:
Maybe this is clearer and more direct? The other one sounds like it could have been you that did something wrong, when in reality it's just that someone else did something.

```suggestion
    'Unleash - Scheduled changes can no longer be applied';
```